conf/application.conf

```diff
@@ -171,6 +171,8 @@ urls {
   hmrcPrivacy = "http://localhost:8503/help/privacy"
   govUk = "https://www.gov.uk"
   pptRegistrationsInfoLink = "https://www.gov.uk/government/publications/introduction-of-plastic-packaging-tax/plastic-packaging-tax"
+  signOut = "http://localhost:9553/bas-gateway/sign-out-without-state"
```

Comment:
Why do we need to call the `bas-gateway`? Can't we just redirect to the `too many email passcode attempts` page with a new session, like we do when signing the user out: https://github.com/hmrc/plastic-packaging-tax-registration-frontend/blob/master/app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/SignOutController.scala#L41?

Reply:
I'm not saying that what you've done here is wrong. I just want to understand whether it's necessary.

Reply:
Probably working on my previous project with bas-gateway made me choose to implement this way :) . I will change it to this approach.